dubbo-registry/dubbo-registry-api/src/main/java/org/apache/dubbo/registry/integration/RegistryProtocol.java

```diff
@@ -214,7 +215,7 @@ private void registerStatedUrl(URL registryUrl, URL registeredProviderUrl, boole
         // decide if we need to delay publish
         boolean register = providerUrl.getParameter(REGISTER_KEY, true);
         if (register) {
-            register(registryUrl, registeredProviderUrl);
+            registry.register(registeredProviderUrl);
```

Comment:
Could you please describe why you change this line

Reply:
I believe the extracted **register** method  is useless, for two reasons: 
1. the **register** method only been referred here(line 217  in RegistryProtocol.java file);
2. **register** method's logic have been implemented in **export** method,eg: get **registry** by **registryUrl**, and register ProviderUrl by **registry** object. But the **registry** object have been obtained in the context of **export** method(line 212).

Reply:
got it, shall we remove `private void register(URL registryUrl, URL registeredProviderUrl)` is this pr? This method seems is no longer needed.